src/driver/landing_pages.mli

```diff
@@ -1,5 +1,18 @@
+type u = unit
+
 open Odoc_unit
 
+val make_index :
+  dirs:dirs ->
+  rel_dir:Fpath.t ->
+  ?libs:(Packages.t * Packages.libty) list ->
+  ?pkgs:Packages.t list ->
+  ?index:index ->
+  enable_warnings:bool ->
+  content:(Format.formatter -> u) ->
+  u ->
```

Comment:
The last unit argument is unnecessary.

Reply:
it was there because all the args are either optional or named. I've removed it and made all args named now.